api/hypershift/v1beta1/hosted_controlplane.go

```diff
@@ -303,6 +303,9 @@ type HostedControlPlaneStatus struct {
 
 	// NodeCount tracks the number of nodes in the HostedControlPlane.
 	NodeCount *int `json:"nodeCount,omitempty"`
+
+	// OldestKubeletVersion tracks the oldest kubelet version in a hosted cluster
```

Comment:
This must be behind the featuregate marker and we will want the same feature gated field within the the HostedCluster

Reply:
I'm curious what you're imagining the data flow to be. `kubelet versions -> HostedCluster -> HostedControlPlane`, or `kubelet versions -> HostedControlPlane -> Hosted Cluster` or `kubelet versions -> HostedCluster ; kubelet versions - > HostedControlPlane`

Reply:
I think I'm confused why we need to involve the HostedCluster object at all here. HCP has the client connection and has the oldestKubeletVersion field, so it can validate the minimumKubeletversion and set a condition on the HCP object. It seems to me that when HCP condition is set, that will bubble up to the HC object which will then get exposed. Why do we need an additional field + additional controller for that field and potentially duplicate the checks